src/plots/plot.ts

```diff
@@ -437,56 +434,49 @@ export class Plot extends Component {
    */
   protected _updateExtents() {
     this._resetEntityStore();
-    this._attrBindings.each((_, attr) => this._updateExtentsForAttr(attr));
-    this._propertyExtents.each((_, property) => this._updateExtentsForProperty(property));
     this._scales().forEach((scale) => scale.addIncludedValuesProvider(this._includedValuesProvider));
   }
 
-  private _updateExtentsForAttr(attr: string) {
-    // Filters should never be applied to attributes
-    this._updateExtentsForKey(attr, this._attrBindings, this._attrExtents, null);
-  }
-
-  protected _updateExtentsForProperty(property: string) {
-    this._updateExtentsForKey(property, this._propertyBindings, this._propertyExtents, this._filterForProperty(property));
-  }
-
   protected _filterForProperty(property: string): IAccessor<boolean> {
     return null;
   }
 
-  private _updateExtentsForKey(key: string, bindings: d3.Map<Plots.IAccessorScaleBinding<any, any>>,
-                               extents: d3.Map<any[]>, filter: IAccessor<boolean>) {
-    const accScaleBinding = bindings.get(key);
-    if (accScaleBinding == null || accScaleBinding.accessor == null) {
-      return;
-    }
-    extents.set(key, this.datasets().map((dataset) => this._computeExtent(dataset, accScaleBinding, filter)));
-  }
-
-  private _computeExtent(dataset: Dataset, accScaleBinding: Plots.IAccessorScaleBinding<any, any>, filter: IAccessor<boolean>): any[] {
-    const accessor = accScaleBinding.accessor;
-    const scale = accScaleBinding.scale;
-
-    if (scale == null) {
-      return [];
-    }
-
-    let data = dataset.data();
-    if (filter != null) {
-      data = data.filter((d, i) => filter(d, i, dataset));
+  protected getExtentsForAttr(attr: string) {
+    if (this._attrExtents[attr] == null) {
```

Comment:
Nope, this should be defined the same way for all attrs (and will always be defined that way)